src/panels/lovelace/editor/config-elements/hui-entities-card-editor.ts

```diff
@@ -174,93 +143,90 @@ export class HuiEntitiesCardEditor extends LitElement
             ></ha-switch>
           </ha-formfield>
         </div>
+        <div class="side-by-side">
```

Comment:
I think these should not be side by side, but both on their own row

Reply:
hmm they dont really take up that much room... but sure